graph-support/src/main/kotlin/com/autonomousapps/graph/DominanceTreeWriter.kt

```diff
@@ -63,6 +63,12 @@ public class DominanceTreeWriter<N : Any>(
 
     /** String representation of [node] that will be printed to console. */
     public fun toString(node: N): String
+
+    /** Total size of the node */
+    public fun getTreeSize(node: N): Long?
+
+    /** The size of the node itself */
+    public fun getSize(node: N): Long?
```

Comment:
Why do these two functions return nullable values?

Reply:
`treeSize` is not calculated in lead nodes currently, and is hence nullable. But I guess I can default it to `size`.
`size` itself is also nullable though because it gets a value out of a map: `files[nodes]?.` ([link](https://github.com/autonomousapps/dependency-analysis-gradle-plugin/pull/1158/files#diff-9b0b98a87dd28fc6a5a168c6b4c53c15d98d3034e88791b0703683098d38cb39R78)) Should I just default to 0 there?